lib/iris/experimental/ugrid.py

```diff
@@ -0,0 +1,75 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the BSD license.
+# See LICENSE in the root of the repository for full licensing details.
+
+"""Legacy import location for mesh support.
+
+See :mod:`iris.ugrid` for the new, correct import location.
+
+Notes
+-----
+This import path alios is provided for backwards compatibility, but will be removed
+in a future release :  Please re-write code to import from the new module path.
+
+This legacy import module will be removed in a future release.
+N.B. it does **not** need to wait for a major release, since the former API was
+experimental.
+
+.. deprecated:: 3.10
+    All the former :mod:`iris.experimental.ugrid` modules have been relocated to
+    :mod:`iris.ugrid` and its submodules.  Please re-write code to import from the new
+    module path.
+    This import path alios is provided for backwards compatibility, but will be removed
+    in a future release : N.B. removing this does **not** need to wait for a major
+    release, since the former API was experimental.
+
+"""
```

Comment:
What is the point of applying this to new files, when we haven't applied it to all the existing ?

Reply:
Because it's a pattern that we'll be adopting and enforced by the `ruff` pre-commit hook.

But no matter, we can enforce that another time outside this PR ... I just thought we could start that journey here knowing that's the destination. No worries 👍